tasktiger/__init__.py

```diff
@@ -276,7 +277,8 @@ def delay(self, func, args=None, kwargs=None, queue=None,
         Queues a task. See README.rst for an explanation of the options.
         """
 
-        if self.config['ALWAYS_EAGER']:
+        if self.config['ALWAYS_EAGER'] and not \
+                (when and when.replace(tzinfo=pytz.UTC) >= datetime.datetime.now(tz=pytz.UTC)):
```

Comment:
This was just to get it to work. I will make the comparison a bit smarter (not to replace the timezone if there already is one set)